### DISCUSSION TITLE
why objbench good then bench

### DISCUSSION THREAD

**Discussion by user-GitHub-user** [juicedata/juicefs#3005] [Q&A] [answered]
juicefs objbench --storage hdfs --access-key hadoop nn1:8020,nn2:8020 --threads 4
![image](https://user-images.githubusercontent.com/18843801/203208773-95c1d1df-8e0d-455e-be11-3e4fb5253ef6.png)

juicefs objbench --storage hdfs --access-key hadoop nn1:8020,nn2:8020 --threads 4 --small-objects 400
![image](https://user-images.githubusercontent.com/18843801/203208633-02ca2a9c-4bb0-4b43-a0e7-fcfa7d118843.png)

Why does objbench seem to perform better when it's also 4 threads uploading 400 small files?

**Comment by zhijian-pro** [comment, marked as answer]
This is a bug.
Fix by https://github.com/juicedata/juicefs/pull/3100
